README.md

```diff
@@ -0,0 +1,140 @@
+## 🏡 Welcome to Accommodation Booking App 🏠
+*****
+![img_house.jpg](img%2Fimg_house.jpg)
+*****
+This app is a booking accommodation application where people can advertise the places they want to rent out and where people looking for accommodations can search for them.  
```

Comment:
```suggestion
This app is a booking accommodation application where people who are looking for accommodations can search for them.  
```
In our case managers are employees. Not some users. We don't have time to make it as we discussed before